go.mod

```diff
@@ -115,7 +115,7 @@ require (
 	kubevirt.io/api v1.1.1
 	kubevirt.io/client-go v1.1.1
 	kubevirt.io/containerized-data-importer-api v1.57.0-alpha1
-	kubevirt.io/kubevirt v1.1.1
+	kubevirt.io/kubevirt v1.3.0
```

Comment:
Since we go.mod to kubevirt v1.3.0, don't we have to bump the chart to v1.3.0 like https://github.com/harvester/harvester/pull/6137?

Reply:
@WebberHuang1118 It's not prepared yet. So, we keep this as is.